cmake/modules/FindPTex.cmake

```diff
@@ -27,6 +27,8 @@ if (WIN32)
             "$ENV{PROGRAMFILES}/Ptex/include"
             /usr/include
             DOC "The directory where Ptexture.h resides")
+    # TODO: figure out a way to add windows support for PTEX_PREFER_STATIC
+    # (since both static and dynamic libs can have a ".lib" suffix)
```

Comment:
Yeah, I suspect something upstream (on the Ptex side) may be the only way to handle this - since I'm not even sure the libraries can / do exist side-by-side, like they do for OSX/Linux.

Or maybe it's just less of an issue on Windows - perhaps there the convention is that you simply have two completely separate builds for static + dynamic, and you would simply set PTEX_LOCATION accordingly?